install/agones/README.md

```diff
@@ -0,0 +1,83 @@
+# Agones
+
+[Agones](https://agones.dev) is designed as a batteries-included, open-source, dedicated game server hosting and scaling project built on top of Kubernetes, with the flexibility you need to tailor it to the needs of your multiplayer game.
+
+## Introduction
+
+This chart install the Agones application and defines deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+
+- [Helm](https://docs.helm.sh/helm/) package manager
+- Kubernetes 1.9+
+- Role-based access controls (RBAC) activated
+- MutatingAdmissionWebhook admission controller activated, see [recommendation](https://kubernetes.io/docs/admin/admission-controllers/#is-there-a-recommended-set-of-admission-controllers-to-use)
+
+## Installing the Chart
+
+To begin working with Helm, run the command:
+```bash
+$ helm init
+```
+
+To install the chart with the release name `my-release`:
```

Comment:
Do we need a section on updating releases? Or not for this PR?

Reply:
I think there is enough work on this PR.

Reply:
SGTM